### ISSUE TITLE
Make all desktop environments available for hana boards

### ISSUE THREAD

**Issue by funkymokey** [ading2210/shimboot#202]
**Is your feature request related to a problem? Please describe.**
After some testing with another user in this repository, I can confirm that only Gnome works on hana Chromebooks. It's a bit frustrating because Gnome is a lot more heavy compared to the other desktops.
**Describe the solution you'd like**
I wish for the other environments to be able to be bootable other than Gnome.
**Describe alternatives you've considered**
I used my own build, along with the user I engaged with. And I can safely determine that only Gnome works.
**Additional context**
404

**Comment by goober3853** [comment]
Wayland or Xorg? What kernel version? Have you confirmed graphics acceleration to be working?

**Comment by ading2210** [comment]
Does KDE wayland work? 

> Yes. Running the glxinfo | grep "direct rendering" command in the terminal gives an output of "Yes"

This doesn't actually mean anything. Even if direct rendering (DRI) is enabled, software rendering might still be used.

**Comment by funkymokey** [comment]
>Does KDE wayland work?

No. Because the sddm manager fails to actually load

> This doesn't actually mean anything. Even if direct rendering (DRI) is enabled, software rendering might still be used.

How can I truly confirm it?

**Comment by es-j3** [comment]
Try a lightweight Wayland session, like sway or weston.

**Comment by ading2210** [comment]
> No. Because the sddm manager fails to actually load

You can configure SDDM to launch in Wayland instead of X11: https://wiki.archlinux.org/title/SDDM#Wayland

**Comment by es-j3** [comment]
> After some testing with another user in this repository, I can confirm that only Gnome works.
Only GNOME works? Or only wayland login managers work?

**Comment by funkymokey** [comment]
> Only GNOME works? Or only wayland login managers work?

Only the GNOME desktop and it's login manager 
If I use tasksel to install a different desktop, then use a different display manager other than gdm3 it boots into a black screen

**Comment by funkymokey** [comment]
> You can configure SDDM to launch in Wayland instead of X11: https://wiki.archlinux.org/title/SDDM#Wayland

I'll test this on monday

**Comment by funkymokey** [comment]
Using tty, I was able to finally boot into something that isint pure black. I now am able to see the cursor, which I believe is progress.

**Comment by funkymokey** [comment]
> Try a lightweight Wayland session, like sway or weston.

@es-j3 Weston worked well. And apps work as well by launching them in terminal.

**Comment by es-j3** [comment]
You could probably trade in your Chromebook to them and get a dedede, or an octopus, or maybe even a corsola.

**Comment by es-j3** [comment]
Obviously, Shimboot doesn't look like a good option for your chromebook. The next best thing you can do atp is unenroll or OOBESCAPE.

**Comment by TheTruePerson** [comment]
> > You could probably trade in your Chromebook to them and get a dedede, or an octopus, or maybe even a corsola.
> 
> I only attempted to use Hana because it's an arm board, and their battery lasts quite a long time, not to mention the CPU has a better base clock speed.
> 
> I could just trade another random guy in my school for their Chromebook, but I have confirmed that only Hana, Grunt, Octopus, and Zork are used in the school.

Well, an octupus would be significantly better
